.github/workflows/integration-test-iOS14_4.yaml

```diff
@@ -18,6 +18,7 @@ jobs:
       LC_CTYPE: en_US.UTF-8
       LANG: en_US.UTF-8
       ABLY_ENV: sandbox
+      TEST_OBSERVABILITY_SERVER_AUTH_KEY: ${{ secrets.TEST_OBSERVABILITY_SERVER_AUTH_KEY }}
```

Comment:
Injecting the secret into the environment [at `job` level](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv) means that it's available to all steps under that job, which is a leak risk - especially when those steps use third party actions.

Please move this to the specific step that needs it - that is, [at `step` level](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsenv).

(same comment applies to other workflows modified in this way in this PR)

Reply:
Ah, thanks for the info. Changed it.